src/main/fc/runtime_config.h

```diff
@@ -60,7 +60,8 @@ typedef enum {
     ARMING_DISABLED_GPS             = (1 << 17),
     ARMING_DISABLED_RESC            = (1 << 18),
     ARMING_DISABLED_RPMFILTER       = (1 << 19),
-    ARMING_DISABLED_ARM_SWITCH      = (1 << 20), // Needs to be the last element, since it's always activated if one of the others is active when arming
+    ARMING_DISABLED_CRASH_DETECTED  = (1 << 20),
```

Comment:
I would suggest moving before `ARMING_DISABLED_THROTTLE` as the user is still likely to have the throttle raised and the "THROTTLE" arming disabled message will have higher priority and display instead.

Reply:
I tried to avoid having to add yet another iteration of this list to all places that keep track of it. But alas!